core/sql/common/Int64.cpp

```diff
@@ -136,6 +140,22 @@ void convertInt64ToAscii(const Int64 &src, char* tgt)
   strcpy(tgt, s);
 }
 
+void convertUInt64ToAscii(const UInt64 &src, char* tgt)
+{
+  UInt64 temp = src;
+  char buffer[21];
+  char *s = &buffer[21];
+  *--s = '\0';
+  do {
+    char c = (char) (temp % 10);
+    if (c < 0)
+      c = -c;
```

Comment:
And there is your cut-and-paste issue :)
This method is a copy of existing method convertInt64ToAscii
which converts a signed number to ascii.

The new convertUInt64ToAscii method doesn't need the 'c < 0' check though it will
never be reached. Will fix that for a later checkin.

Existing convert methods in Int64.cpp pass in input value as reference 
parameter so the new method just followed the same.
Am not sure what the original intent was, maybe some perf improvement.
I will leave it as a reference param.